lib/matplotlib/font_manager.py

```diff
@@ -1275,6 +1279,12 @@ def _findfont_cached(self, prop, fontext, directory, fallback_to_default,
                 best_font = font
             if score == 0:
                 break
+        
+        if best_font is not None:
+            if abs(map_weight_name_to_score(prop.get_weight()) 
```

Comment:
pep8 is pretty unhappy about the formatting here.

Reply:
Thanks! This should be better.